### PR TITLE
replace texts with tooltips

### PR DESCRIPTION
**Pull request by madeuz**
closes CRJI/EIC#602

### CODE REVIEW

File: src/components/document/Tags.js [hunk 1]
```diff
@@ -242,83 +244,110 @@ function Tags({ toolbarButtons }) {
...
+                    </Grid>
+
+                    <Grid item style={{ marginLeft: 20 }}>
+                        <Tooltip
+                            interactive
+                            classes={{ tooltip: classes.noMaxWidth }}
+                            title={
+                                <>
+                                    <Typography variant="body2">
+                                        Tags are made of lowercase ASCII letters, digits, and symbols:{' '}
```
Comment (gabriel-v) [inline src/components/document/Tags.js:335]: Let's move the actual texts (or JSX elements I guess) into a separate file called "help".

For this section, we should link here: https://github.com/liquidinvestigations/docs/wiki/User-Guide:-Hoover#tags 

We can put it a final line in the tooltip:`See more <a>here</a>.`

File: src/components/search/Search.js [hunk 2]
```diff
@@ -333,6 +338,25 @@ export default function Search({ collections }) {
                                                     />
                                                 </Grid>
 
+                                                <Grid item style={{ marginLeft: 20, marginBottom: 7 }}>
+                                                    <Tooltip
+                                                        interactive
+                                                        classes={{ tooltip: classes.noMaxWidth }}
+                                                        title={
+                                                            <>
+                                                                <Typography variant="body2">Enter to search, Shift+Enter for a new line.</Typography>
```
Comment (gabriel-v) [inline src/components/search/Search.js:347]: Same as above; please move these into a separate file.

**Review by gabriel-v — CHANGES_REQUESTED** [review]

File: src/components/document/Tags.js (hunk 1 above)
Comment (madeuz) [reply src/components/document/Tags.js]: done

File: src/components/search/Search.js (hunk 2 above)
Comment (madeuz) [reply src/components/search/Search.js]: done

**Review by gabriel-v — APPROVED** [review]
lgtm, merge after rebasing

**Pull request merged**